package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "scripts": {
+    "test": "cd ./website && ./test-links.sh"
```

Comment:
there's already a `package.json` file in `website/package.json` – would it make more sense to add this `test` script there?

Reply:
I did create a `test` script there as well.  This was created so one could run `yarn test` in either directory and it would work. Kind of a lazy person's script lol.  I can remove it if you like.